README.md

```diff
@@ -40,6 +40,29 @@ resource "env0_configuration_variable" "in_a_template" {
 }
 ```
 
+### How to get VCS credentials for Creating a template or a VCS environment
```

Comment:
Can we please put it below the `Authentication` section? I think this is less common and I don't think we need it to stand out

Reply:
[moving to Authentication](https://github.com/env0/terraform-provider-env0/pull/589/commits/a484d41cd15209da5a2f766b04bf54d75f971140)